OpenSim/Simulation/Model/PrescribedForce.cpp

```diff
@@ -42,14 +42,13 @@ using namespace std;
 /**
  * Default constructor.
  */
-PrescribedForce::PrescribedForce(Body* body)
+PrescribedForce::PrescribedForce(const PhysicalFrame* frame)
 {
```

Comment:
I've been switching passed in dependencies as const references so that there is not confusion about the the Component taking ownership. Although not backward compatible it would it more consistent with other Components (e.g. joints, bushings, etc.. that depend on PhysicalFrames)